Jenkinsfile

```
@@ -7,11 +7,13 @@ class Globals {
     // constants
     static final String PROJECT = 'pyflexplot'
     static final String IMAGE_REPO = 'docker-intern-nexus.meteoswiss.ch'
-    static final String IMAGE_NAME = 'docker-intern-nexus.meteoswiss.ch/flexpart-cosmo/pyflexplot'
+    static final String IMAGE_NAME = 'docker-intern-nexus.meteoswiss.ch/numericalweatherpredictions/dispersionmodelling/flexpart-ifs/pyflexplot'
```

I suggest that since we need the same image of pyflexplot for both COSMO/IFS flexparts, we should not include this in the image name. It can just come under dispersionmodelling
```suggestion
    static final String IMAGE_NAME = 'docker-intern-nexus.meteoswiss.ch/numericalweatherpredictions/dispersionmodelling/pyflexplot'
```